posts/Davide/2017-03-02-hibernate-ogm-5-1-Final-released.adoc

```diff
@@ -0,0 +1,93 @@
+= "Hibernate OGM 5.1 Final is out"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Good news!
+
+We released http://hibernate.org/ogm/[Hibernate OGM] 5.1 Final!
+
+Compared to the latest candidate release, there are not many changes. You can have a look at the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=22351[5.1.0.Final changelog]
+to get all the details.
+
+== What's the difference with 5.0 Final?
+
+I'm glad you asked, this version:
+
+* uses http://hibernate.org/orm/[Hibernate ORM] 5.1
+* supports Infinispan Remote via the http://infinispan.org/docs/stable/user_guide/user_guide.html#hot_rod_protocol[Hot Rod protocol]
+* supports Neo4j remote via the http://boltprotocol.org[Bolt] protocol and the http://neo4j.com/docs/rest-docs/current[Http interface]
+* uses Hibernate Search 5.6 (http://in.relation.to/2017/01/30/hibernate-search-5-6-0-Final-and-5-7-0-CR1[it works with Elasticsearch])
```

Comment:
s/it works/, which works/

and

s/uses Hibernate Search/can integrate with Hibernate Search/